analyzer/src/main/java/org/jboss/gm/analyzer/alignment/AlignmentTask.java

```diff
@@ -81,16 +81,16 @@
  * (whether it's a single or multi module project)
  */
 public class AlignmentTask extends DefaultTask {
-
-    public static final String INJECT_GME_START = "buildscript { apply from: \"gme.gradle\" }";
-    public static final String INJECT_GME_START_KOTLIN = "buildscript { project.apply { from(\"gme.gradle\") } }";
-    public static final String INJECT_GME_END = "apply from: \"gme-pluginconfigs.gradle\"";
-    public static final String INJECT_GME_END_KOTLIN = "project.apply { from(\"gme-pluginconfigs.gradle\") }";
     public static final String GME = "gme.gradle";
+    public static final String INJECT_GME_START = "buildscript { apply from: \"" + GME + "\" }";
+    public static final String INJECT_GME_START_KOTLIN = "buildscript { project.apply { from(\"" + GME + "\") } }";
+    public static final String GME_PLUGINCONFIGS = "gme-pluginconfigs.gradle";
+    public static final String INJECT_GME_END = "apply from: \"" + GME_PLUGINCONFIGS + "\"";
+    public static final String INJECT_GME_END_KOTLIN = "project.apply { from(\"" + GME_PLUGINCONFIGS + "\") }";
     public static final String GRADLE = "gradle";
     public static final String GME_REPOS = "gme-repos.gradle";
-    public static final String APPLY_GME_REPOS = "buildscript { apply from: new File(buildscript.getSourceFile().getParentFile(),\"gme-repos.gradle\"), to: buildscript }";
-    public static final String GME_PLUGINCONFIGS = "gme-pluginconfigs.gradle";
+    public static final String APPLY_GME_REPOS = "buildscript { apply from: new File(buildscript.getSourceFile().getParentFile(), "
+            + GME_REPOS + "\"), to: buildscript }";
```

Comment:
This has broken the injection. Please revert these changes

Reply:
There are constants defined right there that aren't used. If the constants are ever changed, the script texts will refer to the wrong file. I just tried to replace the text there with the constant representing the exact same text, unless I made a typo somewhere.

Also, if something is failing, how can all the tests pass?

Reply:
I see, there's a missing `"`. But there must be a test that injects and then checks that the injection is there? If not, I could add one.